roles/internal/activemq/tasks/install.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: Clean activemq path
+  file:
+    state: absent
+    path: "{{ activemq_install_root }}/activemq"
+
+- name: Download and unarchive activemq
+  unarchive:
```

Comment:
This also needs to be made idempotent somehow. Something like this would work: 

```yaml
- name: Download and unarchive activemq
  unarchive:
    remote_src: yes
    src: http://archive.apache.org/dist/activemq/{{ activemq_version }}/apache-activemq-{{ activemq_version }}-bin.tar.gz
    dest: "{{ activemq_install_root }}"
    owner: "{{ activemq_user }}"
    group: "{{ activemq_group }}"
    creates: "{{ activemq_install_root }}/activemq"
```